multiqc/modules/rna_seqc/rna_seqc.py

```diff
@@ -107,15 +108,66 @@ def rnaseqc_general_stats (self):
             'scale': 'YlGn',
             'modify': lambda x: float(x) * 100.0
         }
+        headers['Expression Profiling Efficiency'] = {
```

Comment:
Expression Profiling Efficiency is a little bit different from % exonic because it is related to the library size, but some people prefer to work with expression efficiency..